### PR TITLE
SAK-31787 Make whole roster printable

### PR DESCRIPTION
**Pull request by adrianfish**

### CODE REVIEW

File: reference/library/src/morpheus-master/sass/modules/tool/roster/_roster.scss.bak [hunk 1]
```diff
@@ -0,0 +1,242 @@
+##{$namespace}sakai-roster {
```
Comment (buckett) [inline reference/library/src/morpheus-master/sass/modules/tool/roster/_roster.scss.bak]: Should this backup file be in the PR?

Comment (adrianfish) [inline reference/library/src/morpheus-master/sass/modules/tool/roster/_roster.scss.bak]: Of course. It is essential. Not really - I was in a rush. Quite the refactor this was.

Comment (adrianfish) [inline reference/library/src/morpheus-master/sass/modules/tool/roster/_roster.scss.bak]: Quick off the mark, Mr Buckett.

Comment (adrianfish) [inline reference/library/src/morpheus-master/sass/modules/tool/roster/_roster.scss.bak]: Sorted.

**Comment by adrianfish** [comment]
Basically, this needs just pulling in and testing on a QA server. It's pretty hard to tell by a code review how it's going to work. If it breaks in QA, I'll fix it.

**Comment by juanjmerono** [comment]
https://jira.sakaiproject.org/browse/SAK-31787?focusedCommentId=211867&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-211867

**Pull request merged**